docs/content/utilities/validation.mdx

```diff
@@ -0,0 +1,292 @@
+---
+title: Validation
+description: Utility
+---
+
+import Note from "../../src/components/Note"
+
+This utility provides JSON Schema validation for payloads held within events and response used in AWS Lambda.
+
+**Key features**
+* Validate incoming events and responses
+* Built-in validation for most common events (API Gateway, SNS, SQS, ...)
+* JMESPath support validate only a sub part of the event
+
+## Install
+
+To install this utility, add the following dependency to your project.
+
+```xml
+<dependency>
+    <groupId>software.amazon.lambda</groupId>
+    <artifactId>powertools-validation</artifactId>
+    <version>0.5.0-beta</version>
+</dependency>
+```
+
+## Validating events
+
+You can validate inbound and outbound events using `@Validation` annotation.
+
+You can also use the `Validator#validate()` methods, if you want more control over the validation process such as handling a validation error.
+
+We support JSON schema version 4, 6, 7 and 201909 (from [jmespath-jackson library](https://github.com/burtcorp/jmespath-java)).
+
+### @Validation annotation
+
+`@Validation` annotation is used to validate either inbound or functions' response.
```

Comment:
`@Validation` annotation is used to validate either inbound or functions' response.

to

`@Validation` annotation is used to validate either inbound events or functions' response.